src/components/Modal.vue

```diff
@@ -35,7 +35,7 @@
 				Indra Kupferschmid, Bram Stein, Nick Sherman, David Jonathan
 				Ross, Koen Kivits, Kenneth Ormandy, Zach Leatherman, Mike
 				“Pomax” Kamermans, John Hudson, Robin Rendle, Pascal
-				Widdershoven and Dave Crossland 💖
+				Widdershoven, Dave Crossland and Amelia Bellamy-Royds 💖
```

Comment:
This PR can't be for me since you didn't use an Oxford comma here.

Reply:
Hey, it's open source, why don't you submit a PR 😝